### PR TITLE
update consensus and committee informatin for at end of every epoch

### PR DESCRIPTION
**Pull request by chaosma**
- update consensus and committee information at end of every epoch
- fix bls library source bug
- local test passed for new node join

### CODE REVIEW

File: scripts/setup_bls_build_flags.sh [hunk 1]
```diff
@@ -35,10 +35,3 @@ case $OS in
       export DYLD_FALLBACK_LIBRARY_PATH=$LD_LIBRARY_PATH
       ;;
 esac
```
Comment (LeoHChen) [inline scripts/setup_bls_build_flags.sh]: why remove this?

Comment (chaosma) [reply scripts/setup_bls_build_flags.sh]: source scripts/setup_bls_build_flags.sh   will fail if not removing this

Comment (chaosma) [reply scripts/setup_bls_build_flags.sh]: I found this bug when I start a new node, because need to source bls library

Comment (LeoHChen) [reply scripts/setup_bls_build_flags.sh]: It can be fixed using "-v" I think.  Don't just remove it.

File: node/node_syncing.go [hunk 2]
```diff
@@ -235,7 +235,9 @@ SyncingLoop:
 				node.Consensus.BlocksNotSynchronized()
 			}
 			node.stateSync.SyncLoop(bc, worker, false)
-			if node.NodeConfig.Role() == nodeconfig.ExplorerNode {
+			// update the consensus and committee information at the end of epoch including explorer node
```
Comment (rlan35) [inline node/node_syncing.go]: This is not enough right? This update happens only after SyncLoop finishes (which means it only happens after it's sync'ed, not during the sync)

**Comment by chaosma** [comment]
Jenkins, test this please

File: node/node_syncing.go (hunk 2 above)
Comment (chaosma) [reply node/node_syncing.go]: Right. I changed it into syncLoop, I think now it should be good

**Pull request merged**